src/app/compare/compare.component.js

```diff
@@ -194,6 +209,7 @@ function CompareCtrl($scope, toaster, dimCompareService, dimStoreService, D2Stor
       if (item.stats) {
         item.stats.forEach(bucketStat);
         bucketStat(item.primStat);
+        bucketStat({ statHash: 0, value: Number(item.dtrRating) });
```

Comment:
I'm pretty sure the rating should already be a number?

Reply:
Had a problem during coding I thought it was because of this, in the end it was something else and forgot to remove this casting.